cv/qc-cv.obo

```diff
@@ -391,6 +390,128 @@ is_a: QC:4000010 ! ID free
 is_a: QC:4000026 ! environment metric
 relationship: has_relation QC:4000013 ! QC metric relation: one run
 
+[Term]
+id: QC:4000069
+name: Fraction of peptide IDs around max MS1
+def: "Fraction of all peptides identified at least X min earlier or later than max MS1 for ID. The values are given in a corresponding list, where the first value is the X minutes before (negative) or after (positive) max MS1 and the second value is the fraction of IDs." [PSI:QC]
+comment: Estimates very early peak broadening.
+is_a: QC:4000005 ! corresponding lists
+is_a: QC:4000009 ! ID based
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+
+[Term]
+id: QC:4000070
+name: Interquartile RT period for peptide identifications
+def: "The interquartile retention time period, in seconds, for all peptide identifications over the complete run." [PSI:QC]
+comment: Longer times indicate better chromatographic separation.
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+
+[Term]
+id: QC:4000071
+name: Peptide identification rate of the interquartile RT period
+def: "The identification rate of peptides for the interquartile retention time period, in peptides per second." [PSI:QC]
+comment: Higher rates indicate efficient sampling and identification.
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+relationship: has_relation QC:4000070 ! Interquartile RT period for peptide identifications
+
+[Term]
+id: QC:4000072
+name: Median MS1 peak FWHM for peptides
+def: "Median of all MS1 peak widths at half maximum (FWHM) for all identified peptide, in seconds" [PSI:QC]
+comment: Sharper peak widths (i.e. smaller values) indicate better chromatographic resolution.
+xref: value-type:xsd\:double "The allowed value-type for this CV term."
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation MS:1000086 ! full width at half-maximum
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+
+[Term]
+id: QC:4000073
+name: Interquartile distance of MS1 peak FWHM for identifications
+def: "Interquartile distance of all MS1 peak widths at half maximum (FWHM) for all identifications, in seconds" [PSI:QC]
+comment: Tighter distributions indicate better consistency.
+xref: value-type:xsd\:double "The allowed value-type for this CV term."
+is_a: QC:4000003 ! single value
+is_a: QC:4000009 ! ID based
+is_a: QC:4000022 ! chromatogram metric
+relationship: has_relation MS:1000086 ! full width at half-maximum
+relationship: has_relation QC:4000013 ! QC metric relation: one run
+
+[Term]
+id: QC:4000074
+name: Median MS1 peak FWHM for peptides in RT decile
```

Comment:
`decile` -> `quantile`?

Reply:
I made the metric more general than @mwalzer suggested at first. Anyway, thanks, corrected it.